infra/cifuzz/actions/action.yml

```diff
@@ -0,0 +1,12 @@
+# action.yml
+name: 'build-fuzzers'
```

Comment:
nit: I think we should drop `-fuzzers` prefix in cifuzz commands, just use `build` and `run` in all scripts, etc

Reply:
We decided that there are other actions that a project would use in its workflow called 'build', so we are going to keep 'build-fuzzers' name.

Reply:
Makes sense, thanks for clarifying!